lib/mix/tasks/dogma.ex

```diff
@@ -11,8 +11,11 @@ defmodule Mix.Tasks.Dogma do
     {dir, formatter} = argv |> parse_args
     config = Config.build
 
+    {:ok, events} = GenEvent.start_link([])
```

Comment:
Would it make any sense for the linting to still be successful if something crashes?